### PR TITLE
Testing To Do list: part 1

### PR DESCRIPTION
**Pull request by jellywiz**
### Project requirements

- This is a pair-programming project. Please decide whose project will you use for testing.
- Use jest framework for testing.
- Create a test file (`[..].test.js`) for a file containing the **add item** and **delete item**  functions that you want to test.
- Take a look at each of these functions. Are they *pure functions*? If the answer is "yes" then writing tests for them should be straightforward. Some of those functions however will update *localStorage* and manipulate the *DOM*. For those, you will need to use mocks:
  - Mock a storage object to "imitate" localStorage operations (only if you are using `jest < 24.x`, in newer versions localStorage is mocked automatically).
  - Mock HTML to test if add/delete functions add or remove exactly one `<li>` element to/from the list in the DOM.
- Make sure you group your tests using the `describe()` method.

### CODE REVIEW

File: src/js/todo.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
+/* eslint-disable import/no-unresolved */
 /* eslint-dsibale no-loop-func, no-func-assign, no-class-assign */
+import deleteItem from './deletetodo.js';
```
Comment (ridwanediallo) [inline src/js/todo.js:3]: - [ ] Hey team good job so far, however, your tests are failing please make sure to fix the issue.

Screenshot from 2022-11-09 21-33-47

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi team, 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/todo.js (hunk 1 above)
Comment (ridwanediallo) [reply src/js/todo.js:3]: Okay thank you, for the screenshot, you can please submit for a second review.

Comment (jellywiz) [reply src/js/todo.js:3]: @ridwanediallo i dont know on the dashboard it looks like it already been approved? I am so confused

Comment (ridwanediallo) [reply src/js/todo.js:3]: Oh okay, when was it approved?

Comment (jellywiz) [reply src/js/todo.js:3]: Right after you said require change
Take a look at the exercise 
image

Comment (jellywiz) [reply src/js/todo.js:3]: @ridwanediallo

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @jellywiz , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**